types/iron/index.d.ts

```diff
@@ -0,0 +1,103 @@
+// Type definitions for iron 5.0
+// Project: https://github.com/hueniverse/iron
+// Definitions by: Simon Schick <https://github.com/simonschick>
+//                 Rafael Souza Fijalkowski <https://github.com/rafaelsouzaf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+/// <reference types="node" />
+
+/**
+ * iron is a cryptographic utility for sealing a JSON object using symmetric key encryption with message integrity
+ * verification. Or in other words, it lets you encrypt an object, send it around (in cookies, authentication
+ * credentials, etc.), then receive it back and decrypt it. The algorithm ensures that the message was not tampered
+ * with, and also provides a simple mechanism for password rotation.
+ * [See docs](https://github.com/hueniverse/iron)
+ */
+declare module 'iron' {
+
+    /**
+     * saltBits - the size of the salt (random buffer used to ensure that two identical objects will generate a different encrypted result.
```

Comment:
Please move the docs to the correct properties.

Reply:
Separated, like that?

```
    /**
     * the size of the salt (random buffer used to ensure that two identical objects will generate a different encrypted result.
     */
    saltBits: number;
    /**
     * the algorithm used ('aes-256-cbc' for encryption and 'sha256' for integrity are the only two supported at this time).
     */
    algorithm: string;
```